aiida/cmdline/params/options/commands/computer.py

```diff
@@ -109,6 +109,15 @@ def should_call_default_mpiprocs_per_machine(ctx):  # pylint: disable=invalid-na
     'Use 0 to specify no default value.',
 )
 
+DEFAULT_MEMORY_PER_MACHINE = OverridableOption(
+    '--default-memory-per-machine',
+    prompt='Default amount of memory per machine (kB).',
+    cls=InteractiveOption,
```

Comment:
Just make it required false, so `None` (i.e. not specifying the option at all) is also accepted
```suggestion
    cls=InteractiveOption,
    required=False,
```

Reply:
I tried this, and the behaviour was exactly the same as if I wouldn't.

Reply:
Yes if you use interactive mode it will prompt, but that is what it is supposed to do. When you pass `--non-interactive` though, it won't prompt and will simply assign `None`... as it should.